js/homework.js

```diff
@@ -0,0 +1,106 @@
+ to changeToBlue when it's clicked. class .jumbotron was selected and assigned as box. donateButton,
+ volunteerButton and volunteerText were assigned to relevant variable.*/
+var blueButton = document.querySelector('#blueBtn');
+blueButton.addEventListener('click', changeToBlue);
+var box = document.querySelector('.jumbotron');
+var donateButton = document.querySelector(".btn.btn-primary.btn-lrg");
+var volunteerButton = document.querySelector(".btn.btn-secondary.btn-lrg");
+var volunteerTextColor = document.querySelector('.volunteerText');
+
+/* This is a function called "changeToBlue" which will change box's,donateButton's 
+, volunteerButton's background color and volunteerTextcolor.*/
+function changeToBlue() {
+    box.style.backgroundColor = '#588fbd';
+    donateButton.style.backgroundColor = '#ffa500';
+    volunteerButton.style.backgroundColor = 'black';
+    volunteerTextColor.style.color = 'white';
+}
+
+//  Making changes when clicking orange button.*/
+var orangeButton = document.querySelector('#orangeBtn');
+orangeButton.addEventListener('click', changetoOrange);
+
+/*changetoOrange function. box, donateButton,volunteerButton's 
+background color changed.*/
+function changetoOrange() {
+    box.style.backgroundColor = '#f0ad4e';
+    donateButton.style.backgroundColor = '#5751fd';
+    volunteerButton.style.backgroundColor = '#31b0d5';
+}
+
+// Making changes when clicking green button.
+var greenButton = document.querySelector('#greenBtn');
+greenButton.addEventListener('click', changeToGreen);
+
+/* changeToGreen function.box, donateButton,volunteerButton's 
+background color changed. */
+function changeToGreen() {
+    box.style.backgroundColor = '#87ca8a';
+    donateButton.style.backgroundColor = 'black';
+    volunteerButton.style.backgroundColor = '#8c9c08';
+}
+
+/* Selecting the submit button and creating "validateTheForm" function.*/
+var submitButton = document.querySelector('#submitButton');
+submitButton.addEventListener('click', validateTheForm);
+
+// This is ValidateTheForm function. used event.preventDefault() method. 
+function validateTheForm(event) {
+    event.preventDefault();
+
+    // selecting namebox with document.querySelector.
+    var namebox = document.querySelector('#example-text-input');
+    // Adding value in namebox.
+    var nameboxWV = namebox.value;
+
+    // selecting email input box with document.querySelector.
+    var emailInput = document.querySelector('#exampleInputEmail1');
+    // Adding value in emailInput.
+    var emailInputWV = emailInput.value;
+
+    // selecting describe yourself box with document.querySelector.
+    var describeYourselfInput = document.querySelector('#exampleTextarea');
+    // Adding value in emailInput.
+    var describeYourselfInputWV = describeYourselfInput.value;
+
+    // Setting the form is Valid by using boolean.(true)
+    var formIsValid = true;
+    /* if the namebox with value.length is invalid, changed background color of box to red 
```

Comment:
Long comments tend to indicate that a part of the code can be replaced by an inner function. 

The nameboxMv if statement can be replaced by a function like "isNameValid" returning true or false depending on the case.

You can apply the same for email and description. This reduces this function complexity and removes the need of comments.

I will elaborate more in the next comment.